main.tf

```diff
@@ -0,0 +1,22 @@
+module "eip" {
+  source = "github.com/zzxwill/terraform-alicloud-eip"
```

Comment:
Will update this repo to Alibaba Cloud official repo once this PR is approved https://github.com/terraform-alicloud-modules/terraform-alicloud-eip/pull/9.